src/sid/msm.py

```diff
@@ -141,7 +140,7 @@ def _msm(
     flat_empirical_moments = _flatten_index(empirical_moments)
     flat_simulated_moments = _flatten_index(simulated_moments)
 
-    moment_errors = flat_empirical_moments - flat_simulated_moments
+    moment_errors = flat_simulated_moments - flat_empirical_moments
```

Comment:
Is the order important to manfred? If it's true, I would add the comment.
```suggestion
    # Order is important to manfred.
    moment_errors = flat_simulated_moments - flat_empirical_moments
```

Reply:
Yes, the order is important if we want to exploit that infection probabilities have a positive effect on infection numbers.